Day7n8/github-api-project-ts/src/index.tsx

```diff
@@ -34,9 +34,9 @@ const myAppRouter = createBrowserRouter([
 ]);
 
 root.render(
-  <React.StrictMode>
-    <RouterProvider router={myAppRouter} />
-  </React.StrictMode>
+  // <React.StrictMode>
```

Comment:
We can remove this commented code if you don't need it for your documenting purpose.